DataFormats/HcalRecHit/interface/HBHERecHit.h

```diff
@@ -22,6 +22,9 @@ class HBHERecHit : public CaloRecHit {
   /// get the id
   inline HcalDetId id() const { return HcalDetId(detid()); }
 
+  inline void setChiSquared(const float chi2) {chiSquared_ = chi2;}
+  inline float getChiSquared() const {return chiSquared_;}
```

Comment:
please change to ```chiSquared``` to preserve the style with all other access methods, which do not have a "get" prefix.